python/pyspark/ml/tests/test_algorithms.py

```diff
@@ -83,7 +83,7 @@ def test_raw_and_probability_prediction(self):
         result = model.transform(test).head()
         expected_prediction = 2.0
         expected_probability = [0.0, 0.0, 1.0]
-        expected_rawPrediction = [-11.824, -8.298, 22.5299]
+        expected_rawPrediction = [-11.6081922998, -8.15827998691, 22.17757045]
         self.assertTrue(result.prediction, expected_prediction)
         self.assertTrue(np.allclose(result.probability, expected_probability, atol=1E-4))
         self.assertTrue(np.allclose(result.rawPrediction, expected_rawPrediction, rtol=0.1))
```

Comment:
BTW, @zhengruifeng, if the changes in the main codes are correct, we could just simply increase `rtol` to  something like `0.2` too.

Reply:
I will leave it to you @zhengruifeng

Reply:
3.2 is coming, let's revert it for now.

Reply:
👌